doc/source/examples/extended_examples/hpc/hpc_ml_ga.rst

```diff
@@ -0,0 +1,447 @@
+        mapdl.et(1, "BEAM188")
+        mapdl.keyopt(1, 4, 1)  # transverse shear stress output
+
+        # Material properties
+        mapdl.mp("EX", 1, 2e7)  # N/cm2
+        mapdl.mp("PRXY", 1, 0.27)  # Poisson's ratio
+
+        # Beam properties in centimeters
+        sec_num = 1
+        mapdl.sectype(sec_num, "BEAM", "I", "ISection", 3)
+        mapdl.secoffset("CENT")
+        mapdl.secdata(15, 15, 29, 2, 2, 1)  # dimensions are in centimeters
+
+        # Setting FEM model
+        mapdl.n(1, 0, 0, 0)
+        mapdl.n(12, 110, 0, 0)
+        mapdl.n(23, 220, 0, 0)
+        mapdl.fill(1, 12, 10)
+        mapdl.fill(12, 23, 10)
+
+        for node in mapdl.mesh.nnum[:-1]:
+            mapdl.e(node, node + 1)
+
+        # Define the boundary conditions
+        # Allow movement only in the X and Z direction
+        for const in ["UX", "UY", "ROTX", "ROTZ"]:
+            mapdl.d("all", const)
+
+        # constrain just nodes 1 and 23 in the Z direction
+        mapdl.d(1, "UZ")
+        mapdl.d(23, "UZ")
+
+        # apply a -Z force at node 12
+        mapdl.f(12, "FZ", force[0])
+
+        # run the static analysis
+        mapdl.run("/solu")
+        mapdl.antype("static")
+        mapdl.solve()
+
+        # Extracting data
+        UZ = mapdl.post_processing.nodal_displacement("Z")
+        UZ_node_12 = UZ[12]
+
+        return UZ_node_12
+
+As it can be seen, this function returns the control parameter for this model, which is the displacement at the Z-direction on the node 12 (``UZ_node_12``).
+
+
+Setting MAPDL pool
+==================
+
+The genetic algorithms are expensive regarding the amount of calculation needed to reach an optimal solution. As seen before, many simulations must be performed to select, crossover, and mutate
+all the chromosomes across all the populations.
+For this reason, to speed up the process, it is desirable to have as many MAPDL instances as possible, so
+each one can calculate one chromosome fit function.
+
+To manage multiple MAPDL instances, the best approach is to use :py:class:`~ansys.mapdl.core.pool.MapdlPool` class.
+
+.. code-block:: python
+
+    from ansys.mapdl.core import MapdlPool
+
+    # Starting pool
+    # Number of instances should be equal to number of CPUs
+    # as set later in the ``sbatch`` command
+    pool = MapdlPool(n_instances=10)
+    print(pool)
+
+
+Define deflection target
+========================
+
+Because this is a demonstration example, the target displacement is calculated 
+using the beam function itself using a force of 22840 :math:`N/cm^2`.
+
+.. code-block:: python
+
+    # Calculate target displacement
+    mapdl = pool[0]
+    force = 22840  # N/cm2
+    target_displacement = calculate_beam(pool[0], [force])
+    print(f"Setting target to {target_displacement} for force {force}")
+
+
+Genetic algorithm model
+=======================
+
+Introduction
+------------
+
+The library `PyGAD <pygad_docs_>`_ is used to configure the genetic algorithm.::
```

Comment:
```suggestion
You use the `PyGAD <pygad_docs_>`_ library to configure the genetic algorithm.::
```
Will the ".::" used at the end of the sentence render in the documentation correctly? I think it will render as ".:" instead of concluding only with the period.

Reply:
It should be rendered the text after as quotes or something like that.